app/controllers/users_controller.rb

```diff
@@ -0,0 +1,10 @@
+class UsersController < ApplicationController
+  def index
+    @users = User.all
+  end
+
+  def show
+    @user = User.find(params[:id])
+    @posts = @user.posts
+  end
```

Comment:
- [ ] Kindly use the `most_recent_posts` to display posts inside `User#show` instead of passing all posts to match the wireframe.